insights/specs/default.py

```diff
@@ -305,8 +306,6 @@ def docker_container_ids(broker):
     grub1_config_perms = simple_command("/bin/ls -l /boot/grub/grub.conf")  # RHEL6
     grub2_cfg = simple_file("/boot/grub2/grub.cfg")
     grub2_efi_cfg = simple_file("boot/efi/EFI/redhat/grub.cfg")
-    grub2_efi_grubenv = simple_file("/boot/efi/EFI/redhat/grubenv")
-    grub2_grubenv = simple_file("/boot/grub2/grubenv")
     grubby_default_index = simple_command("/usr/sbin/grubby --default-index")  # only RHEL7 and updwards
```

Comment:
@xiangce do the above 2 specs work correctly, it seems from the multiple issues that maybe the parser did not work.  If they were working correctly then we should deprecate the specs and parsers instead of removing them.  If they did not work correctly then it is OK to remove them.

Reply:
@bfahr - Yes, these two specs are just added for a new rule. They didn't work as excepted and was removed from the rule. Thanks.